index.bs

```diff
@@ -474,6 +462,59 @@ To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [
         extensions, and I think it'll be more difficult to ship them after inverting the
         error-handling behavior.
 
+To perform a <dfn abstract-op>cross-origin resource policy check</dfn> given a [=request=]
+(|request|) and a [=response=] (|response|), run these steps:
+
+1.  Let |embedder policy| be |request|'s [=request/client=]'s
+    [=environment settings object/embedder policy=].
+
+2.  If |request|'s [=request/reserved client=] is not `null`, then set |embedder policy|
+    to a new [=/embedder policy=].
+
+3.  If |embedder policy|'s [=embedder policy/report only reporting endpoint=] is not `null` and the
+    result of running [$cross-origin resource policy internal check] with
+    [=embedder policy/report only value=], |request| and |response| is `blocked`, then run these
+    steps:
+
+    1. Let |blocked url| be the first URL of |request|'s [=request/URL list=].
```

Comment:
Nit: `|request|'s [=request/URL=]` is a shorter way of pointing to the first URL of the URL list.

Reply:
Done.